http4s/src/main/scala/com/avast/grpc/jsonbrige/http4s/Http4s.scala

```diff
@@ -18,8 +18,11 @@ import scala.language.higherKinds
 
 object Http4s extends StrictLogging {
 
-  def apply[F[_]: ToTask](configuration: Configuration)(bridges: GrpcJsonBridge[F, _ <: BindableService]*)(
-      implicit sch: Scheduler): HttpService[IO] = {
+  def apply[F[_]: Sync](configuration: Configuration)(bridges: GrpcJsonBridge[F, _ <: BindableService]*)(
+      implicit sch: Scheduler): HttpService[F] = {
+    implicit val h: Http4sDsl[F] = Http4sDsl[F]
+    import h._
+
     val services = bridges.map(s => (s.serviceName, s): (String, GrpcJsonBridge[F, _])).toMap
```

Comment:
It's confusing to use `bridge` and `service` for the same thing. Please unify it, e. g. rename this local variable to `bridgesByName`.